lib/src/lib/table/components/table/table.component.html

```diff
@@ -66,6 +77,7 @@
             <lab900-action-button *ngFor="let action of tableActionsBack" [data]="element" [action]="action"></lab900-action-button>
           </div>
         </td>
+        <td mat-footer-cell *matFooterCellDef></td>
```

Comment:
shouldn't this have an `*ngIf`, to hide it in case there are no footers

Reply:
done